client/src/js/actions/AppActions.js

```diff
@@ -0,0 +1,229 @@
+import AppDispatcher from '../dispatcher/appDispatcher';
+import AppConstants from '../constants/appConstants';
+
+const AppActions = { 
+    saveContact(contact){
+        AppDispatcher.handleViewAction({
+            actionType: AppConstants.SAVE_CONTACT,
+            contact
+        })
+
+    },
+
```

Comment:
I think you should be using 2 spaces for indentation not 4...except if the style guide has changed.

Also use `saveContact(contact) {`